src/spark_rapids_ml/core.py

```diff
@@ -751,53 +751,3 @@ def predict_udf(iterator: Iterator[pd.DataFrame]) -> Iterator[pd.Series]:
     def _out_schema(self, input_schema: StructType) -> Union[StructType, str]:
         assert self.dtype is not None
         return dtype_to_pyspark_type(self.dtype)
-
-
-def _set_pyspark_cuml_cls_param_attrs(
```

Comment:
Is this not used?

Reply:
Not used any more since I made the cuml params changes.  Passed `run_tests.sh` and `run_benchmarks.sh`.